front/cypress/e2e/survey_builder/create_and_fill_survey.cy.ts

```diff
@@ -134,19 +119,15 @@ describe.skip('Survey builder', () => {
     cy.visit(
       `admin/projects/${projectId}/phases/${phaseId}/native-survey/edit`
     );
-    cy.get('[data-cy="e2e-short-answer"]').click();
+    waitForCustomFormFields();
+    cy.get('[data-cy="e2e-linear-scale"]').click();
```

Comment:
I used a few different types of fields across the tests, rather than always using short answer as example